requirements.txt

```diff
@@ -1,10 +1,10 @@
-uproot>=4.0.4
+uproot>=4.1.1
 matplotlib>=3.3.4
-pandas==1.1.5
+pandas>=1.1.5
 scikit-learn>=0.24.1
-xgboost==1.3.3
+xgboost>=1.3.3
 shap==0.38.1
 bayesian-optimization==1.2.0
-pyarrow==3.0.0
-ipython==7.16.1
+pyarrow>=3.0.0
+ipython>=7.16.1
 jedi==0.17.2
```

Comment:
For `pyarrow` and `xgboost` we can bump to the latest version `pyarrow 5.0.0` and `xgboost 1.4.2`. For `pandas` and `ipython` using >= has the effect of having different versions between python 3.6 and 3.7/3.8, maybe it is not what we want.

Reply:
I updated the two packages. For the pandas and ipython issue: python 3.6 is phasing out at the end of the year, we can live with this situation for a couple of months, considering that we anyway know personally the bulk of the users and most of them are on modern platforms.